src/share/util/shr_pio_mod.F90

```diff
@@ -177,7 +182,18 @@ subroutine shr_pio_init2(comp_id, comp_name, comp_iamin, comp_comm, comp_comm_ia
 
     if(pio_async_interface) then
 #ifdef PIO1
-       call pio_init(total_comps,mpi_comm_world, comp_comm, io_comm, iosystems, rearr_opts=pio_rearr_opts)
+       call pio_init(total_comps,mpi_comm_world, comp_comm, io_comm, iosystems)
```

Comment:
But I think you are right, we should be able to get rid of the ifdef now because pio_set_rearr_opts() should work for async cases too (Although pio_init() does not
work for async+pio2).